test/functional/helpers.lua

```diff
@@ -401,6 +401,11 @@ function module.retry(max, max_ms, fn)
   end
 end
 
+-- Calls fn() until it succeeds.
+function module.retry_inf(fn)
+  module.retry(nil, nil, fn)
```

Comment:
would rather not introduce retry_inf(). Instead retry() could behave like this if it has exactly 1 arg that is a function.

Reply:
Sure. I assume I just need to flip the arguments so that `fn` is first?